site/docs/v1/tech/apis/identity-providers/samlv2-idp-initiated.adoc

```diff
@@ -34,7 +34,24 @@ The following values will likely be required by your SAML v2 IdP Initiated Ident
 
 These values are autogenerated and viewable within the UI after creating the Identity Provider. They can be viewed by navigating to [breadcrumb]#Settings -> Identity Providers -> SAMLv2 IdP Initiated -> View#.
 
-include::docs/v1/tech/identity-providers/_saml-config-urls.adoc[]
+`<base_url>` is the URL for your FusionAuth instance, something like `\https://login.piedpiper.com`.
```

Comment:
Is this different than what is in the `_saml-config-urls.adoc`?

Reply:
The difference is the ACS urls which can change depending on the idp-init toggle and we need to address with https://github.com/FusionAuth/fusionauth-issues/issues/2146, and when that happens this doc will need updated again as well